docs/apis/scala_shell.md

```diff
@@ -30,15 +30,16 @@ Flink and setting up a cluster please refer to
 To use the shell with an integrated Flink cluster just execute:
 
 ~~~bash
-bin/start-scala-shell.sh 
+bin/start-scala-shell.sh local
 ~~~
 
 in the root directory of your binary Flink directory.
 
-To use it with a running cluster you can supply the host and port of the JobManager with:
+To use it with a running cluster start the scala shell with the keyword remote
```

Comment:
Adding comma and quotation marks would be helpful to make readable like following:
```
To use it with a running cluster, start the scala shell with the keyword "remote"
```